packages/hooks/src/useWithdrawalFiatAddress.ts

```diff
@@ -0,0 +1,40 @@
+import { useCallback, useEffect } from 'react';
+import { useRequest } from '@deriv/api';
+import { useStore } from '@deriv/stores';
+
+const useWithdrawalFiatAddress = () => {
+    const { client } = useStore();
+    const { ui } = useStore();
+    const { is_dark_mode_on } = ui;
+    const { data, mutate, ...rest } = useRequest('cashier');
+    const withdrawal_address =
+        typeof data?.cashier === 'string' ? `${data?.cashier}&DarkMode=${is_dark_mode_on ? 'on' : 'off'}` : undefined;
+
+    const send = useCallback(() => {
+        if (client.verification_code.payment_withdraw)
```

Comment:
```suggestion
```
@aum-deriv This hook doesn't need to be aware of the `verification_code` in the client store, We only render the component which calls this hook after checking that `verification_code` exists.